core/src/main/java/com/linecorp/armeria/internal/logging/ContentPreviewingUtil.java

```diff
@@ -164,7 +164,12 @@ private static String sanitize(
             BiFunction<? super RequestContext, String,
                     ? extends @Nullable Object> contentSanitizer,
             RequestContext ctx, String produced) {
-        final Object sanitized = contentSanitizer.apply(ctx, produced);
+        Object sanitized = null;
+        try {
+            sanitized = contentSanitizer.apply(ctx, produced);
```

Comment:
Should we widen the try-catch block of the call sites?
```
try {
    produced = responseContentPreviewer.produce();
    if (produced != null) {
        produced = sanitize(contentSanitizer, ctx, produced);
    }
} catch (Exception e) {
    logger.warn("Unexpected exception while producing the response content preview. " +
                "previewer: {}", responseContentPreviewer, e);
}
```

Reply:
In this case, `contentPreview` will be `null` so I applied try-catch inside the `sanitize` to be `<sanitized>`.
How do you think? Which one is better?

Reply:
I don’t see `sanitize` is allowed to throw an exception.
If an exception is raised, we may assume that it is not sanitized.

Reply:
> we may assume that it is not sanitized

be produced content preview or null?

Reply:
How about using `produded`? This is because it is not sanitized.